stripe.go

```diff
@@ -158,6 +169,7 @@ type BackendImplementation struct {
 	//
 	// See also SetNetworkRetriesSleep.
 	networkRetriesSleep bool
+	lastRequestMetrics  *RequestMetrics
```

Comment:
Because `networkRetriesSleep` above has its own comment, could you just add a newline above this one just to show that it's not part of the same group?